hugo/internal/github_hugo_theme.bzl

```diff
@@ -8,12 +8,23 @@ filegroup(
 )
 """
 
-def github_hugo_theme(name, owner, repo, commit, **kwargs):
-    url = "https://github.com/{owner}/{repo}/archive/{commit}.zip".format(
-        owner = owner,
-        repo = repo,
-        commit = commit,
-    )
+DEFAULT_GITHUB_HOST = "github.com"
+
+def github_hugo_theme(name, owner, repo, commit, github_host=None, **kwargs):
```

Comment:
Just do `github_host=DEFAULT_GITHUB_HOST` in the params... Makes the if-else un-necessary

Reply:
Thanks for that, I've sent new commit with this change.